test/views/html/ContributionsEarningsSpec.scala

```diff
@@ -42,9 +42,9 @@ class ContributionsEarningsSpec extends GmpViewSpec {
     }
 
     "have a table with td's" in {
```

Comment:
Assuming this should be "have table with th's" given what's been changed underneath?

Reply:
This is fixed now